src/ex2/UIManager.js

```diff
@@ -0,0 +1,92 @@
+class UIManager {
+  constructor(){
+  this.sortBtn = document.getElementById("sortBtn");
+  this.footerElement = document.getElementById("footertId");
+  this.clearAllBtn = document.getElementById("clearAllBtnId");
+  this.pendingTasksCount = document.getElementById("pendingTasksCountId");
+  this.todoList = document.getElementById("listElement");
+  this.EventListenersArray = []
+  this.idCounter = 2000
+  }
+ /*  static sortBtn = document.getElementById("sortBtn");
+  static footerElement = document.getElementById("footertId");
+  static clearAllBtn = document.getElementById("clearAllBtnId");
+  static pendingTasksCount = document.getElementById("pendingTasksCountId");
+  static todoList = document.getElementById("listElement"); */
+
+  showButtonsAndFooter() {
+    this.showSortBtn();
+    this.showFooter();
+    this.showClearAllBtn();
+  }
+
+  showSortBtn() {
+    this.sortBtn.classList.remove("inactive");
+    this.sortBtn.classList.add("active");
+  }
+
+  showFooter() {
+    this.footerElement.classList.remove("inactive");
+    this.footerElement.classList.add("active");
+  }
+
+  showClearAllBtn() {
+    this.clearAllBtn.classList.remove("inactive");
+    this.clearAllBtn.classList.add("active");
+  }
+
+  UIHandleAddRenderItem(textItem, itemListLength) {
+    this.pendingTasksCount.innerText = itemListLength;
+    const todoLi = document.createElement("li");
+    todoLi.classList.add("todo");
+    todoLi.innerText = textItem;
+    const trashButton = document.createElement("button");
+    trashButton.innerHTML = "🗑️";
+    trashButton.classList.add("trashBtn");
+   
+    trashButton.setAttribute('id',`${this.idCounter}TrashID` )
+    this.idCounter++
+ 
+    todoLi.appendChild(trashButton);
+    this.todoList.appendChild(todoLi);
+   
+    
+    
+  return 
```

Comment:
no need to return if you're not returning anything in this case. also mind the number of empty lines.